NightscoutServiceKit/Extensions/StoredDosingDecision.swift

```diff
@@ -35,12 +35,19 @@ extension StoredDosingDecision {
     }
     
     var recommendTempBasal: RecommendedTempBasal? {
-        guard let tempBasalRecommendationDate = tempBasalRecommendationDate else {
+        guard let recommendedTempBasal = recommendedTempBasal else {
```

Comment:
[nit] Ooh, this was hard to see that it wasn't an infinite loop; that _this_ variable is named `recommendTempBasal`, but it is calling `recommendedTempBasal`.  The old names were actually a little clearer to me.

Reply:
I renamed everything so it is clear.